Final Project/Assets/RenderPipeline/SecondaryRay/GetTriangleNormal.compute

```diff
@@ -0,0 +1,15 @@
+﻿#ifndef GETTRIANGLENORMAL_COMPUTE
```

Comment:
You can get normal directly from the triangle like 
_Triangles[index].normal
or you can get it from Rayhit.normal
You might don't need to use this fuction.

Reply:
true. thanks!